provider/testdata/data/postcard/fingerprints.csv

```diff
@@ -813,6 +813,462 @@ props/gc@1, und, 16984B, 4055bc27ac6b3a4ee6e0a4fb1914a52ffd4de5ce9ea6ba65895e8db
+relativetime/long/year@1, tr, 83B, b78a1123adf95ba979614b38bed1f113cb74e72cfd19da1a14b5e12c83b690ec
+relativetime/long/year@1, und, 59B, f2ad6a3cb7229aecf6e13fec4eaa713cb6b086c0c33884e88c2676af9da260ee
+relativetime/narrow/day@1, ar, 198B, fbab704bcb291209d8480303a2a95383a1aecfdc62cddbf2e55d2cac452aa189
+relativetime/narrow/day@1, ar-EG, 198B, fbab704bcb291209d8480303a2a95383a1aecfdc62cddbf2e55d2cac452aa189
+relativetime/narrow/day@1, bn, 169B, baf5333b9a2a3e0370db4fed653f45d7de8e2a79565dababf4dcf668151e4572
+relativetime/narrow/day@1, ccp, 400B, b9b351ed49db3fa41a289b3b4065e764409fb92bb49142b3f715fbdb70236ab9
+relativetime/narrow/day@1, en, 70B, 509846aa8e2b87c2f011aef86a3677468f40b821de8776cfcd22caf1e66e4501
+relativetime/narrow/day@1, en-001, 70B, 509846aa8e2b87c2f011aef86a3677468f40b821de8776cfcd22caf1e66e4501
+relativetime/narrow/day@1, en-ZA, 70B, 509846aa8e2b87c2f011aef86a3677468f40b821de8776cfcd22caf1e66e4501
+relativetime/narrow/day@1, es, 87B, 7e18e94f9d4b8cb09b018b11e41f4aee74e5340f48774e72cedbb1d6fc138902
+relativetime/narrow/day@1, es-AR, 95B, 3daf12b935304ccd6c2e5127151f32ef11240c19025dc07f4246194022bd51f0
+relativetime/narrow/day@1, fil, 135B, 0b2e4d7b724ebe4c9b27e5d6c8e1de1954f4d8003c15127fcbdbcdb0ef978522
+relativetime/narrow/day@1, fr, 89B, 4056988c0123b43dc8a69cbddf9211051ddd9e63de273a7537330fe7dc5ae40f
+relativetime/narrow/day@1, ja, 77B, 53716bec74b8bb7910cd2648bc5a21deed595c67fe8706e7d7a584f93c29178d
+relativetime/narrow/day@1, ru, 145B, c49d5d556e7800963023e58851393a3594bb40997adfb48982a758075141b143
+relativetime/narrow/day@1, sr, 136B, 0823e7f430979a57acda5f9d29e95713117b07fbfa926c07d230f18505196868
+relativetime/narrow/day@1, sr-Cyrl, 136B, 0823e7f430979a57acda5f9d29e95713117b07fbfa926c07d230f18505196868
+relativetime/narrow/day@1, sr-Latn, 94B, cec24faa886382d57b87ee529cf3cc0f25d0a6439bee8896a2552079c98d753b
+relativetime/narrow/day@1, th, 190B, 12ecd7f5f133a216005ac2f34d7c6b46d34d5af4848e0dd3828b5dd1aa56293f
+relativetime/narrow/day@1, tr, 98B, 2f4d6b30daea5df24500d78c8021dc22c6dbe65a92aa4022f1667f1529b56e5a
+relativetime/narrow/day@1, und, 54B, 60d2632bec415afc86666c16bf81e4e7e70c3851be5bb385ace463f89faf07ce
+relativetime/narrow/hour@1, ar, 171B, 70b3532b016f724fbfedf11a45faa70ce8298e637824eace6217e5885b06c550
+relativetime/narrow/hour@1, ar-EG, 171B, 70b3532b016f724fbfedf11a45faa70ce8298e637824eace6217e5885b06c550
+relativetime/narrow/hour@1, bn, 105B, 2300cc3aadb0c291d3becdfa743147ccae378e50815913602be6f936107b20d4
+relativetime/narrow/hour@1, ccp, 138B, 36b6113d2cbacb70007ac61ce222349e848a4150b0845bd9cd9e17d968cd93b1
+relativetime/narrow/hour@1, en, 50B, 2a119b1d4dfa007dbc4d2e4d79cf39bd23b39167399b5878fea8cd6e7269fb5b
+relativetime/narrow/hour@1, en-001, 48B, f15555b8ed2135cb792b46db2aa2b769423fc763fc8bb348ce58a7eff542ee43
+relativetime/narrow/hour@1, en-ZA, 48B, f15555b8ed2135cb792b46db2aa2b769423fc763fc8bb348ce58a7eff542ee43
```

Comment:
Praise: Note how for narrow hour, `en` differs from `en-001`. Since these are small keys, only the difference for narrow hour needs to be stored, and we can keep deduplicating the rest of the time scales that are identical. With a bigger key, a single change like this would "spoil" the deduplication from working most optimally.